pkg/processor/runtime/shell/runtime.go

```diff
@@ -75,9 +75,8 @@ func NewRuntime(parentLogger logger.Logger, configuration *Configuration) (runti
 }
 
 func (s *shell) ProcessEvent(event nuclio.Event, functionLogger logger.Logger) (interface{}, error) {
-	command := s.command
-
-	command += " " + s.getCommandArguments(event)
+	command := []string{s.command}
```

Comment:
this kind of change may cost with performance, can you ensure it provides added value? (e.g. benchmark it)